flink-cdc-e2e-tests/flink-cdc-pipeline-e2e-tests/src/test/java/org/apache/flink/cdc/pipeline/tests/utils/PipelineTestEnvironment.java

```diff
@@ -114,30 +141,42 @@ public static List<String> getFlinkVersion() {
     public void before() throws Exception {
         LOG.info("Starting containers...");
         jobManagerConsumer = new ToStringConsumer();
-
-        String flinkProperties = getFlinkProperties();
-
+        // These cmds will put the FLINK_PROPERTIES to flink-conf.yaml
+        List<String> cmds = new ArrayList<>();
+        cmds.add("cp /opt/flink/conf/flink-conf.yaml /opt/flink/conf/flink-conf.yaml.tmp");
```

Comment:
Seems we can use `GenericContainer#withCopyToContainer(Transferable#of)` to place config file in-place w/o appending startup command